desktop/core/src/desktop/js/sql/reference/flink/reservedKeywords.ts

```diff
@@ -0,0 +1,106 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const RESERVED_WORDS: Set<string> = new Set([
```

Comment:
current reserved keywords taken from general , more to add from https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/sql/#reserved-keywords 
@romainr  ?

Reply:
Yes, and the current keyworkds were just copy pasted so a bunch are invalid actually: https://github.com/cloudera/hue/issues/1672

Reply:
i will take care of it in annother PR, merging this one now.